libs/feature/dataviz/src/lib/geo-table-view/geo-table-view.component.ts

```diff
@@ -1,8 +1,40 @@
 import { Component } from '@angular/core'
+import {
+  FEATURE_COLLECTION_POINT_FIXTURE_4326,
+  MAP_CTX_LAYER_XYZ_FIXTURE,
+  MapContextLayerTypeEnum,
+} from '@geonetwork-ui/ui/map'
+import { fromLonLat } from 'ol/proj'
 
 @Component({
   selector: 'gn-ui-geo-table-view',
   templateUrl: './geo-table-view.component.html',
   styleUrls: ['./geo-table-view.component.css'],
 })
-export class GeoTableViewComponent {}
+export class GeoTableViewComponent {
+  geojson = FEATURE_COLLECTION_POINT_FIXTURE_4326
+
+  get tableData() {
+    return this.geojson.features.map((f) => ({
+      id: f.id,
+      ...f.properties,
+    }))
+  }
+
+  get mapData() {
```

Comment:
this is executed a lot of time, rather init the config in the `ngInit `once

Reply:
maybe if I switch to OnPush it will only happen on input change? using OnInit means we assume the input will not change later on

Reply:
`OnPush` is really expected here right, but I don't think it's enough, put a console log in your getter and you'll understand :)